src/main/java/com/boardgo/domain/meeting/repository/MeetingDslRepositoryImpl.java

```diff
@@ -0,0 +1,237 @@
+package com.boardgo.domain.meeting.repository;
+
+import com.boardgo.domain.boardgame.entity.QBoardGameEntity;
+import com.boardgo.domain.boardgame.entity.QBoardGameGenreEntity;
+import com.boardgo.domain.meeting.controller.request.MeetingSearchRequest;
+import com.boardgo.domain.meeting.entity.MeetingState;
+import com.boardgo.domain.meeting.entity.QMeetingEntity;
+import com.boardgo.domain.meeting.entity.QMeetingGameMatchEntity;
+import com.boardgo.domain.meeting.entity.QMeetingGenreMatchEntity;
+import com.boardgo.domain.meeting.entity.QMeetingParticipantSubEntity;
+import com.boardgo.domain.meeting.repository.projection.MeetingSearchProjection;
+import com.boardgo.domain.meeting.repository.response.MeetingSearchResponse;
+import com.boardgo.domain.user.entity.QUserInfoEntity;
+import com.querydsl.core.BooleanBuilder;
+import com.querydsl.core.Tuple;
+import com.querydsl.core.types.OrderSpecifier;
+import com.querydsl.core.types.Projections;
+import com.querydsl.core.types.dsl.BooleanExpression;
+import com.querydsl.core.types.dsl.Expressions;
+import com.querydsl.jpa.JPAExpressions;
+import com.querydsl.jpa.impl.JPAQueryFactory;
+import jakarta.persistence.EntityManager;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class MeetingDslRepositoryImpl implements MeetingDslRepository {
+    private final JPAQueryFactory queryFactory;
+    private final QMeetingEntity m = QMeetingEntity.meetingEntity;
+    private final QUserInfoEntity u = QUserInfoEntity.userInfoEntity;
+    private final QMeetingGameMatchEntity mg = QMeetingGameMatchEntity.meetingGameMatchEntity;
+    private final QBoardGameEntity b = QBoardGameEntity.boardGameEntity;
+    private final QMeetingGenreMatchEntity mg2 = QMeetingGenreMatchEntity.meetingGenreMatchEntity;
+    private final QBoardGameGenreEntity g = QBoardGameGenreEntity.boardGameGenreEntity;
```

Comment:
엔티티 명을 적거나 엔티티의 카멜케이스를 따서 변수명을 하는게 가독성이 좋을 듯합니다 ㅜㅜ
쿼리문을 읽을때 계속 이게 어떤 엔티티인지 위에서 확인하게 되더라구요

```suggestion
    private final QMeetingGameMatchEntity mgam = QMeetingGameMatchEntity.meetingGameMatchEntity;
    private final QBoardGameEntity bg = QBoardGameEntity.boardGameEntity;
    private final QMeetingGenreMatchEntity mgem = QMeetingGenreMatchEntity.meetingGenreMatchEntity;
    private final QBoardGameGenreEntity bgg = QBoardGameGenreEntity.boardGameGenreEntity;
```

Reply:
```java
  private final QMeetingGameMatchEntity mgm = QMeetingGameMatchEntity.meetingGameMatchEntity;    
  ...
  private final QMeetingGenreMatchEntity mgm = QMeetingGenreMatchEntity.meetingGenreMatchEntity;
```

이 두개의 변수가 같아서 mg2라고 하긴 했습니다 ㅠㅠ 이 때는 어떻게 하시나요 ?